DataFormats/Parameters/src/GRPMagField.cxx

```diff
@@ -41,3 +41,25 @@ void GRPMagField::print() const
 {
   printf("magnet currents (A) L3 = %.3f, Dipole = %.f; uniformity = %s\n", getL3Current(), getDipoleCurrent(), mUniformField ? "true" : "false");
 }
+
+o2::units::Current_t GRPMagField::checkDipoleOverride()
+{
+  static float v = getenv("O2_OVERRIDE_DIPOLE_CURRENT") ? atof(getenv("O2_OVERRIDE_DIPOLE_CURRENT")) : NOOVERRIDEVAL;
+  static bool alarmShown = false;
+  if (v != NOOVERRIDEVAL && !alarmShown) {
+    LOGP(alarm, "Overriding DIPOLE current to {}", v);
```

Comment:
One more point I forgot, I would make this an ERROR, just to be sure it is seen by the shift crew. (I think they usually filter out alarms)

Reply:
ok